controller/model/posture_check_model_process.go

```diff
@@ -42,14 +43,14 @@ func (p *PostureCheckProcess) Evaluate(pd *PostureData) bool {
 			}
 
 			if p.Fingerprint != "" {
-				if p.Fingerprint != process.SignerFingerprint {
+				if !strings.EqualFold(p.Fingerprint, process.SignerFingerprint) {
```

Comment:
same general questions around 'normalizing' fingerprints and hashes. can they have whitespace, can they have separators? if yes should that all be normalized? (maybe it is already)

Reply:
Will address in #400